.gitignore

```diff
@@ -6,3 +6,4 @@ profile.cov
 hack/python-sdk/openapi-generator-cli.jar
 dep-crds/
 dep-manifests/
+vendor/
```

Comment:
why is this necessary?

Reply:
In case of git add . to include the vendor dir.

Reply:
@lowang-bh We don't use vendoring in this repository. So I don't think we need this ignoring. Are there any reasons for this ignore?

Reply:
@tenzen-y sometimes user runs `go mod vendor`.  In case of `git add .` to include it. I can remove it if it is really not need.

Reply:
I'm ok with adding `vendor` to `.gitignore`.